tests/framework/installer/ceph_installer.go

```diff
@@ -210,6 +205,10 @@ func (h *CephInstaller) CreateK8sRookClusterWithHostPathAndDevices(namespace, sy
 		}
 	}
 
+	if err := h.WipeClusterDisks(namespace); err != nil {
+		return fmt.Errorf("failed to wipe cluster disks. %+v", err)
```

Comment:
Do we need to fail the test if the wipe fails? Or can we allow it to attempt to continue? In the future when the tests only deploy OSDs on devices we would need this working, but until then we don't need to block the test, right?

Reply:
We shouldn't proceed if the wipe fails, because the tests won't actually verify disk usage at that point.

Reply:
I ask that after seeing the CI fail on 1.15 [here](https://jenkins.rook.io/blue/rest/organizations/jenkins/pipelines/rook/pipelines/rook/branches/PR-3495/runs/114/nodes/54/steps/120/log/?start=0) (where we don't have devices), but the disk wipe failed. 
```
failed to create cluster mrc-n1. failed to wipe cluster disks. failed to wait for wipe jobs to complete. timed out waiting for the condition
```
Maybe there was something else wrong with the cluster and the multi-cluster job would have failed anyway. We can merge it like this and keep an eye on it.

Reply:
Oh I see. Yeah let's keep an eye on it and an eye on the wipe job's logs when that happens.

Reply:
In the multi-cluster suite, do the tests try to create both clusters at once?

Reply:
The multi-cluster suite creates the two clusters serially. So if the wipe job cleans up every time, this could certainly be an issue for the second cluster. Thus, running the multi cluster test on pvcs would be much better if we can get #4151 finished.